cli/pipeline/run.go

```diff
@@ -185,8 +185,17 @@ func runPipelineStep(pipeline *PipelineDefinition, step *PipelineDefinitionStep,
 				return errors.New(msg)
 			}
 		case <-ctx.Done():
+			pod, _ := pods.Get(podDefinition.PodName, metav1.GetOptions{})
+			reason := "Timeout waiting for pod to start. Cluster might not have sufficient resources."
```

Comment:
nitpick: "Timed out while waiting for pod to start. Cluster might not have sufficient resources."